partner_second_lastname/models/base_config_settings.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingenieria S.L. - Antonio Espinosa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class BaseConfigSettings(models.TransientModel):
+    _inherit = 'base.config.settings'
+
+    def _partner_names_order_selection(self):
+        options = super(
+            BaseConfigSettings, self)._partner_names_order_selection()
+        new_options = []
+        mapping = {
+            'last_first': 'Lastname SecondLastname Firstname',
+            'last_first_comma': 'Lastname SecondLastname, Firstname',
+            'first_last': 'Firstname Lastname SecondLastname',
+        }
+        for option in options:
+            new_option = option
+            if option[0] in mapping:
+                new_option = (option[0], mapping[option[0]])
+            new_options.append(new_option)
```

Comment:
No, because some method in the chain could change the options. We only change label to options that already exists in dictionary, don't want to add any additional option.